client/rest-high-level/src/test/java/org/elasticsearch/client/RollupIT.java

```diff
@@ -158,5 +166,26 @@ public void testPutRollupJob() throws Exception {
                 }
             }
         });
+
+        // TODO when we move cleaning rollup into ESTestCase we can randomly choose the _all version of this request
+        GetRollupJobRequest getRollupJobRequest = new GetRollupJobRequest(id);
+        GetRollupJobResponse getResponse = execute(getRollupJobRequest, rollupClient::getRollupJob, rollupClient::getRollupJobAsync);
+        assertThat(getResponse.getJobs(), hasSize(1));
+        JobWrapper job = getResponse.getJobs().get(0);
+        assertEquals(putRollupJobRequest.getConfig(), job.getJob());
+        assertThat(job.getStats().getNumPages(), lessThan(10L));
+        assertEquals(numDocs, job.getStats().getNumDocuments());
+        assertThat(job.getStats().getNumInvocations(), lessThan(10L));
```

Comment:
Hmm, i wonder if this is potentially flaky?  E.g. the job is configured to trigger every second, so if it takes longer than 10 seconds to get to this point for whatever reason, the assertion might fail.

I think the other assertions are fine since they should be deterministic (number of docs indexed, number of pages, etc)

Reply:
Got it. I hadn't realized it'd count time when it didn't do any work. I'll just assert >0.

Reply:
Yeah, it's not necessarily time... but it increments each time the indexer is "triggered", even if nothing is done after the trigger.  Handy way to help debug if the indexer is "doing things" even if no documents are being emitted for some reason.